package.json

```diff
@@ -61,27 +57,41 @@
     "autoprefixer": "10.4.2",
     "axios": "0.25.0",
     "babel-core": "7.0.0-bridge.0",
-    "babel-jest": "27.5.0",
+    "babel-jest": "27.5.1",
+    "dotenv-safe": "8.2.0",
     "eslint": "8.8.0",
     "eslint-config-prettier": "8.3.0",
     "eslint-plugin-jest": "26.1.0",
     "eslint-plugin-nuxt": "3.1.0",
     "eslint-plugin-prettier": "4.0.0",
-    "eslint-plugin-tailwindcss": "^3.4.3",
+    "eslint-plugin-tailwindcss": "3.4.3",
     "eslint-plugin-vue": "8.4.1",
     "eslint-plugin-vuejs-accessibility": "1.1.1",
     "glob": "7.2.0",
-    "jest": "27.5.0",
-    "jest-environment-jsdom": "27.5.0",
+    "husky": "7.0.4",
+    "jest": "27.5.1",
+    "jest-environment-jsdom": "27.5.1",
     "jest-serializer-vue": "2.0.2",
     "lodash": "4.17.21",
-    "npm-run-all": "4.1.5",
     "postcss": "8.4.6",
     "prettier": "2.5.1",
     "pretty-quick": "3.1.3",
     "tailwindcss": "3.0.19",
     "ts-jest": "27.1.3",
     "ts-node": "10.5.0",
+    "typescript": "4.5.5",
     "vue-jest": "3.0.7"
+  },
+  "pnpm": {
+    "peerDependencyRules": {
+      "ignoreMissing": [
+        "vue",
```

Comment:
We safe to ignore Vue? Does Nuxt "handle" this or something?

Reply:
because nuxt has vue as a dep, missing peer issues isnt an issue.